integrations.yml

```diff
@@ -317,3 +317,10 @@
   website: "https://pullpokes.com"
   icon: "pull-pokes.png"
   category: 5
+-
+  name: "TMetric"
+  description: "TMetric is a work time tracker for freelancers, agencies, and companies."
+  website: "https://tmetric.com/integrations/basecamp-time-tracking/"
+  icon: "tmetric.png"
+  category: 5
```

Comment:
@JordanS22 wouldn't the category be `2 - Time Tracking, Invoicing, and Accounting`?

Reply:
Yep, definitely!